packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -142,21 +136,21 @@ contract OPContractsManager is ISemver, Initializable {
     /// @notice Address of the ProtocolVersions contract shared by all chains.
     IProtocolVersions public immutable protocolVersions;
 
-    /// @notice The latest release of the OP Contracts Manager, as a string of the format `op-contracts/vX.Y.Z`.
-    string public latestRelease;
-
-    /// @notice Maps a release version to a contract name to it's implementation data.
-    mapping(string => mapping(string => Implementation)) public implementations;
+    // @notice L1 smart contracts release deployed by this version of OPCM. This is used in opcm to signal which version
+    // of the L1 smart contracts is deployed. It takes the format of `op-contracts/v*.*.*`.
+    string public l1ContractsReleaseVersion;
```

Comment:
nit, we can shorten to `l1ContractsVersion` or `l1ContractsRelease`

@maurelian any preferenceo on keeping this a string vs. changing to `(uint major, uint minor, uint patch, bool isRc)` to facilitate parsing in solidity?

Reply:
Change made here: https://github.com/ethereum-optimism/optimism/pull/12851/commits/c8068686416d6eda365a44d2bd74c9b33ac736d7#diff-32c676c7a2cc25dc934a436546eccc2c3484b44491cc1569dce382ffb13b84c7R141 

I'll wait to see if there are any other requests on the string vs alternative parsing.

Reply:
Let's just keep it as a string. I am no longer enthusiastic about using versions to determine if it is safe to reinit.